tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -1145,9 +1146,31 @@ describe(__filename, () => {
     expect(root.find('.UserProfileEdit-confirm-button')).toHaveLength(1);
     expect(root.find('.UserProfileEdit-confirm-button').children())
       .toHaveText('Yes, delete my profile');
+    expect(root.find('.UserProfileEdit-confirm-button'))
+      .toHaveProp('disabled', false);
     expect(root.find('.UserProfileEdit-cancel-button')).toHaveLength(1);
   });
 
+  it('disables the confirm button is user has listed add-ons', () => {
```

Comment:
do you mean `..if user..` here (instead of `..is user..`?

Reply:
I meant `if` of course :x